app/models/workarea/payment/refund/credit_card.decorator

```diff
@@ -19,7 +19,10 @@ module Workarea
           customer_profile_id: customer_profile_id,
           customer_payment_profile_id: customer_payment_profile_id,
           amount: refund_amount,
-          trans_id: transaction.reference.response.params['direct_response']['transaction_id']
+          trans_id: transaction.reference.response.params['direct_response']['transaction_id'],
+          order: {
+            invoice_number: tender.payment.id.first(20) # auth net has max length 20 for this field
```

Comment:
Out-of-box, the `Payment#id` field is `Order#id`, and that only has 5 characters. Did you experience a problem with just using `tender.payment.id` here?

Reply:
Oh no, I actually just copy and pasted this straight from https://github.com/workarea-commerce/workarea-authorize-cim/blob/master/app/models/workarea/payment/authorize/credit_card.decorator#L29 and then tested that it worked correctly in our sandbox.authorize.net account